### PR TITLE
[14.0][IMP] helpdesk_mgmt: creation of ticket notified to followers of the team assigned

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
The creation of ticket is notified to followers of the team assigned. This is how it's implemented in Project or Maintenance, so I think it makes sense having it also here.

@ForgeFlow

### CODE REVIEW

File: helpdesk_mgmt/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -125,13 +125,21 @@ def _onchange_dominion_user_id(self):
...
     def create(self, vals):
         if vals.get("number", "/") == "/":
             vals["number"] = self._prepare_ticket_number(vals)
         if not vals.get("team_id") and vals.get("category_id"):
             vals["team_id"] = self._prepare_team_id(vals)
-        return super().create(vals)
+        res = super().create(vals)
+        if res.team_id:
+            res.message_subscribe(
+                partner_ids=res.team_id.message_follower_ids.partner_id.mapped("id")
```
Comment (AaronHForgeFlow) [inline helpdesk_mgmt/models/helpdesk_ticket.py]: I think this is safer

```suggestion
                partner_ids=res.team_id.message_follower_ids.mapped("partner_id.id")
```

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM!

File: helpdesk_mgmt/models/helpdesk_ticket.py (hunk 1 above)
Comment (LoisRForgeFlow) [inline helpdesk_mgmt/models/helpdesk_ticket.py]: Can you test if it is actually needed to do it explicitly?

**Review by LoisRForgeFlow — APPROVED** [review]
Now LGTM :+1:

**Review by pedrobaeza — APPROVED** [review]
Interesting to be back-ported to v13:

/ocabot merge minor

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**